platform/ethereum/domain.go

```diff
@@ -1,14 +1,33 @@
 package ethereum
 
 import (
-	CoinType "github.com/trustwallet/blockatlas/coin"
+	"strings"
+
+	"github.com/trustwallet/blockatlas/coin"
+	"github.com/trustwallet/blockatlas/pkg/address"
 	"github.com/trustwallet/blockatlas/pkg/blockatlas"
 	"github.com/trustwallet/blockatlas/pkg/errors"
 	"github.com/trustwallet/blockatlas/pkg/logger"
 	"github.com/trustwallet/blockatlas/platform/ethereum/ens"
-	AddressEncoder "github.com/trustwallet/ens-coincodec"
+	coincodec "github.com/trustwallet/ens-coincodec"
 )
 
+var domains = map[string]interface{}{
```

Comment:
I think we can define a common string set somewhere
```golang
type StringSet map[string]struct{}
```
struct{} is empty and can't be used incorrectly